sync/src/chain.rs

```diff
@@ -617,7 +633,7 @@ impl ChainSync {
 	}
 
 	/// Find some headers or blocks to download for a peer.
-	fn request_blocks(&mut self, io: &mut SyncIo, peer_id: PeerId) {
+	fn request_blocks(&mut self, io: &mut SyncIo, peer_id: PeerId, ignore_others: bool) {
```

Comment:
Downloading logic has been changed to prevent downloading too many blocks ahead when downloading in parallel. Now instead of downloading a block which is too far away from the lowest block which is currently being downloaded peer will help to download this lowest block.